src/main/resources/templates/in-app/placement-useful-information/v1.0.0.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <title>Deferral</title>
+</head>
+<body>
+  <h1>In-App Notification</h1>
+  <h2>Subject</h2>
+  <th:block th:fragment="subject">Deferral</th:block>
+  <h2>Content</h2>
+  <th:block th:fragment="content">
+    <p>
+      This notification pertains to your placement in
```

Comment:
Sorry, I should have revised this text in line with James' edits. We now have a standard first paragraph that does away with this 'pertains' nonsense 😁 

```
<p>
        For your placement starting
        <b>
          <th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _">
            (start date missing)
          </th:block>
        </b> at
        <b>
          <th:block th:text="${not #strings.isEmpty(site)} ? ${site} : _">
            (site name missing)
          </th:block>
        </b> in
        <b>
          <th:block th:text="${not #strings.isEmpty(specialty)} ? ${specialty} : _">
            (speciality name missing)
          </th:block></b>, please see the below information.
      </p>
```